src/Workspaces/Core/Portable/Classification/Classifier.cs

```diff
@@ -21,9 +21,12 @@ namespace Microsoft.CodeAnalysis.Classification
     public static class Classifier
     {
         internal static PooledObject<SegmentedList<ClassifiedSpan>> GetPooledList(out SegmentedList<ClassifiedSpan> classifiedSpans)
+            => GetPooledList<ClassifiedSpan>(out classifiedSpans);
+
+        internal static PooledObject<SegmentedList<T>> GetPooledList<T>(out SegmentedList<T> classifiedSpans)
```

Comment:
>internal static PooledObject<SegmentedList<T>> GetPooledList<T>(out SegmentedList<T> classifiedSpans) [](http://example.com/codeflow?start=8&length=101)

This doesn't feel specific to ClassifiedSpans anymore. Would it make sense for this to be in a more general location?

Reply:
yes.  will address in followup.